src/g3log/time.hpp

```diff
@@ -45,6 +45,7 @@ namespace g3
    * WARNING: At time of writing there is only so-so compiler support for
    * std::put_time. A possible fix if your c++11 library is not updated is to
    * modify this to use std::strftime instead */
+   std::string localtime_formatted(const timespec &time_snapshot, const std::string &time_format) ;
```

Comment:
I think this is a good idea. Internally the other localtime_formatted will be used but a client could use it if they wanted to. 

When we set the spec for the formatting flag we should document that in this header too since it is an override of the normal formatting for ```std::put_time```